plugins/source/cloudflare/client/spec.go

```diff
@@ -21,5 +25,12 @@ func (s *Spec) SetDefaults() {
 	}
 }
 
+func (Spec) JSONSchemaExtend(sc *jsonschema.Schema) {
+	sc.OneOf = []*jsonschema.Schema{
+		{Required: []string{"api_token"}},
+		{Required: []string{"api_key", "api_email"}},
+	}
```

Comment:
Can we add a test of what happens if both `api_token` and `api_key, api_email` are specified?

Reply:
Done in https://github.com/cloudquery/cloudquery/pull/17605/commits/13db042e0302a9a52943c8d5c30937a20fcea37b